app/blueprints/template/routes.py

```diff
@@ -78,10 +78,19 @@ def view_templates():
     return render_template("view_templates.html", **params, error=error)
 
 
-@template_bp.route("/<form_id>", methods=["GET", "POST"])
+@template_bp.route("/<uuid:form_id>", methods=["GET"])
+def template_details(form_id):
+    """
+    Renders template details page
+    """
+    form = get_form_by_id(form_id)
+    return render_template("template_details.html", form=form)
+
+
+@template_bp.route("/<uuid:form_id>/edit", methods=["GET", "POST"])
```

Comment:
If you're splitting out the GET and POST into different handlers, then this should be POST-only. SonarQube is complaining about a single function handling both GET and POST - if you want to resolve that error (rather than just accepting it in sonarqube, which I'd be fine with), then make this only POST. 👍